### ISSUE TITLE
Restarts twice

### ISSUE THREAD

**Issue by yormi** [kmagiera/babel-watch#42]
Hi there !

First of all, I love what you did :)

Now, I don't know why but when I change a file, babel-watch restarts my app twice. This is pretty fast so it wouldn't be much of a problem if it wasn't for the app not finding a module... (this issue in fact: https://github.com/kmagiera/babel-watch/issues/37)

I thought maybe I have something in my project that is probably messing with `babel-watch` since there's no issues about it here so I made this simple test case:

**index.js**
```javascript
const someText = require('./aFileToRequire')
console.log(someText)
```

**aFileToRequire.js**
```javascript
module.exports = "banana"
```

`babel-watch ./index.js`

When saving either file, it restarts twice.

I tried with node 6.9.2 ans 7.4.0.

Anybody with the same problem ?

**Comment by SystemParadox** [comment]
I also have this issue (babel-watch 2.0.7), on Linux Mint, node 6.14.2.

It doesn't happen with the `-L` option.

**Comment by STRML** [comment]
Fixed in `7.1.0` with restart debouncing.
